aidants_connect_carto_api/utilities.py

```diff
@@ -0,0 +1,190 @@
+import re
+import humanized_opening_hours as hoh
+
+
+# CHOICE fields
+
+
+def find_verbose_choice(choices: tuple, value: str):
+    for choice in choices:
+        if choice[1] == value:
+            return choice[0]
+    return ""
+    # raise Exception(f"error with {value}")
+
+
+# Float fields
+
+
+def process_float(value: str):
+    if value:
+        value = value.replace(",", ".")
+        return float(value)
+    return None
```

Comment:
Pour info ou rappel, une fonction sans `return` renvoie `None` implicitement, donc cette ligne n'est pas strictement nécessaire. Après, ça m'arrive aussi de l'écrire quand même, car [`Explicit is better than implicit`](https://www.python.org/dev/peps/pep-0020/#the-zen-of-python) :)

Reply:
ok! on va rester sur None pour l'instant pour rester explicite, après ces helpers je ne sais pas à quel point ils resteront là dans l'état, c'etait vraiment pour me simplifier l'import du csv

Reply:
Vu qu'on va sûrement devoir écrire pas mal de scripts d'import ad hoc, cette approche un peu "boîte à outils" me parait pertinente — et probablement amenée à durer :)